app/grandchallenge/challenges/templates/challenges/challenge_list.html

```diff
@@ -250,15 +250,15 @@ <h2>{{ year }}</h2>
                     {% for challenge in challenges %}
                         <div class="col-12 col-md-6 col-lg-4 mb-3 projectlink {{ challenge.filter_classes|join:" " }} {{ challenge.year }}">
                             <div class="card" style="height:400px">
-                                <a id="{{ challenge.short_name }}"
+                                <a id="{{ challenge.short_name }}" title="{{ challenge.title }}"
                                    href="{{ challenge.get_absolute_url }}"
                                    style="display:block;">
                                     <div class="card-header small p-0 d-flex flex-row flex-nowrap align-items-center">
                                         <img class="thumbnail rounded-top-left flex-shrink-0"
                                              alt="{{ challenge.short_name }} logo"
                                              src="{{ challenge.thumb_image_url }}">
-                                        <h3 class="m-0 px-3 text-nowrap text-truncate flex-shrink-1">
-                                            {% firstof challenge.title challenge.short_name %}
+                                        <h3 class="m-0 px-3 flex-shrink-1">
+                                            {% firstof challenge.title|truncatewords:3 challenge.short_name|truncatewords:3 %}
```

Comment:
I'm not too sure that we want to do a hard truncate without telling the user. What happens if we keep the text-truncate marker from bootstrap by allowing multiple lines?

Reply:
Sadly, that doesn't work because multi-line wrapping of text is non-trivial in CSS.

Reply:
Also, I'm not really sure how this is different from the current situation as we truncate the title now even more?